src/components/exercise9/fetching.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import UserComponent from "./userComponent";
+
+export default function Fetch() {
+  const [user, setUser] = useState([]);
+  const [random, setRandom] = useState(0);
+
+  useEffect(
+    function () {
+      fetch("https://random-data-api.com/api/users/random_user?size=10")
+        .then((res) => res.json())
+        .then((data) => setUser(data));
+      console.log(user);
+    },
```

Comment:
- Kindly remove the `console.log` from your code. It's a good practice to always clean up your codes by removing 
all the `console.log` and `un-used codes`

- Such commands are likely to slow the execution of the production version, especially commands that write to a disk file (disk operations are just about the slowest action on any computer).

It may only be milliseconds of extra time per console.log command, but those milliseconds do add up, especially if they occur in frequently repeated code (such as looping structures).

Reply:
Thank you for pointing that out and i have removed it